README.md

```diff
@@ -159,7 +159,11 @@ These thresholds help you maintain consistent test quality as your codebase evol
     min-coverage-overall: 80  # Require at least 80% overall coverage
     min-coverage-changed-files: 70  # Require at least 70% coverage for changed files
     min-coverage-per-changed-file: 60  # Require at least 60% coverage for each changed file
-    fail-on-threshold: true  # Fail the GitHub action if a threshold is not reached
+    
+    fail-on-threshold: true  # Fail the GitHub action if defined threshold is not reached
+        or
+    # Fail the GitHub action if overall or changed files average threshold is not reached
+    fail-on-threshold: overall, changed-files-average 
```

Comment:
This is weird.
First it creates the impression that `or` is part of the workflow yaml file.
Second it actually makes the code snippet unusable directly and confusing, as the input `fail-on-threshold` is defined there twice.